media/js/firefox/new.js

```diff
@@ -112,8 +110,12 @@
             state = 'Desktop, Firefox not up-to-date';
         }
     }
-    window._gaq = _gaq || [];
-    window._gaq.push(['_setCustomVar', 4, '/new conditional message', state, 3]);
+    //GA Custom Dimension in Pageview
+    window.dataLayer = window.dataLayer || [];
```

Comment:
Since you're passing in `dataLayer` as a local variable already, there shouldn't be any need for this check.